external-api/data-types.md

```diff
@@ -949,6 +949,8 @@ A booking can belong to an order, but it doesn't have to. An order can include a
 
 Example:
 
+__NOTE:__ _The `payment` in this example is just available for the booking webhooks and __<ins>NOT</ins>__ for this External API. `payment` comes when the selected schema is Bookings + payments for the webhook._
```

Comment:
Extremely minor, but in terms of formatting (and readability in raw Markdown) I don't think this comment necessarily needs to be italicized. But I will defer to you and @alexlewin-fareharbor !

Reply:
I might even say, put a `*` in the schema itself, and then have a matching `*` in front of the note down here.

I also notice that there is already some mention of this, here:

https://github.com/FareHarbor/fareharbor-docs/blob/master/external-api/webhooks.md

So we might include a (relative) link to that page in our "Note". 

Robbie, I'm okay with however you choose to do this, as long as it seems like it will reduce confusion and reduce customers' need to call FareHarbor support!